e2e/npm_translate_lock_link_workspace/foo/BUILD.bazel

```diff
@@ -1,6 +1,6 @@
 load("@bazel_lib//lib:testing.bzl", "assert_contains")
-load("@foo//:@aspect-test/a/package_json.bzl", "bin")
 load("@foo//:defs.bzl", "js_run_binary")
+load("@foo//foo:@aspect-test/a/package_json.bzl", "bin")
```

Comment:
This is where the `package.json` is, you can't change it anymore. I think it was super confusing or maybe buggy that workspace paths (directories containing `package.json`) were being modified based on `root_package`... 🤔